en/12_Project_Governance/05_Release_policy.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Release policy
+summary: Outline the release lifecycle and support commitment for Silverstripe CMS
+icon: code-branch
+---
+
+# Silverstripe CMS release policy
+
+This document outlines the release policy for Silverstripe CMS. It applies to all [commercially supported modules](/project_governance/supported_modules/).
+
+Community modules are not covered by this policy. Modules in the `silverstripe` GitHub organisation that are not commercially supported are updated on a best effort basis.
+
+The purpose of this policy is to provide clear guidance on the types of versions and version releases, including their cadence, scope, and support lifecycle, ensuring predictability and stability for our users and developers.
+
+> [!NOTE]
+> Refer to our [definition of public API](/project_governance/public_api/).
+
+## Verion numbering
+
+### Semantic versioning
+
+Silverstripe CMS adheres to [Semantic versioning](https://semver.org/) for its version numbering of individual modules such as `silverstripe/framework`
```

Comment:
I'm not happy with this section, particularly as it's talking about modules rather than the CMS

However the concept of semantic versioning is core to releases and API/dependencies that are allowed to change within each release, so we probably need to clear definition to provide context?

It gets really confusing because I talk about "CMS versions" below which is what people in the real world actually refer to